schemas/io.snowplow.ecomm/click_item/jsonschema/1-0-0

```diff
@@ -0,0 +1,38 @@
+{
+    "description": "Event occurrs when an item (a product or bundle) is clicked on.",
+    "properties": {
+        "item_id": {
+            "type": "string",
+            "description": "ID reference to the item (as listed in item) that was clicked on.",
+            "maxLength": 255
+        },
+        "item_type": {
+            "type": "string",
+            "enum": ["product", "bundle"],
+            "description": "The general type of the item, currently supports the two types of a singular product and a bundle of products.",
+            "maxLength": 255
+        },
+        "click_position": {
+            "type": [
+                "integer",
+                "null"
+            ],
+            "description": "The numerical position of the item on the page, if the item appears first, click_position would be 1",
```

Comment:
I suppose that the position is based on the concept of a 'collection'. This means you can have multiple collections of products in the same page and all of them have a product with a position of 1.
E.g.
![image](https://github.com/snowplow/iglu-central/assets/15251081/4a1fd662-bb2a-41d0-918b-ac625499c407)


In the case we are talking about a product list type of page this can be based on 'the items on the page' because you have a single collection.

Is my thinking correct here ?

Reply:
You're right, I'll add a `collection_id` field to this schema and have it to be required, so users will know where to find the specific collection from which the item that is clicked is in.